src/services/v0/v0blocks/BlocksService.ts

```diff
@@ -381,8 +382,11 @@ export class BlocksService extends AbstractService {
 	 *
 	 * @param genericCall `GenericCall`
 	 */
-	private parseGenericCall(genericCall: GenericCall): ISanitizedCall {
-		const { sectionName, methodName, callIndex } = genericCall;
+	private parseGenericCall(
+		genericCall: GenericCall,
+		block: Block
```

Comment:
Would it be possible to pass around the `registry` rather than the block? It would clarify the intent a bit (maybe it’s more efficient too?). Without knowing the context I think I would have had to read a fair amount of code before figuring out what the second params is for.

Reply:
I will change to just the registry. I actually did the whole block just for readability and to make it clear the registry is associated with the block